triage.yaml

```diff
@@ -0,0 +1,17 @@
+name: Check Triage Status of Issue
+on: 
+  issues:
+    types: [opened, closed, reopened, transferred, labeled, unlabeled]
+    # Issue is created, Issue is closed, Issue added or removed from projects, Labels added/removed
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update Kanban
+        uses: machine-learning-apps/kubeflow-triage@master
```

Comment:
Should this be changed to kubeflow/kubeflow-triage?
So a GitHubAction step "uses" always points at a GitHub repository not a docker image?

Reply:
I updated this but some additional changes you should be aware of:

- I changed the name of the repo to https://github.com/kubeflow/triage-issues, because it doesn't make sense to have the name kubeflow in the name of the repo.  

- You can refer directly to the docker container, but the interface becomes opaque to end users.  I'll leave it as is, pointing to kubeflow/triage-issues and we can change it later if you want